packages/reactivated/src/errors.tsx

```diff
@@ -0,0 +1,13 @@
+export interface SerializedError {
+    message: string;
+    stack: string | null;
+}
+
+export interface SSRErrorResponse {
+    error: SerializedError;
+}
+
+export const serializeError = (error: Error): SerializedError => ({
+    message: (error as Error).toString(),
```

Comment:
Is this cast here needed? It's already coming in as an error. Minor nitpick, if I find nothing else still good to merge.

eslint isn't currently used on reactivated (it is on the autogenerated projects), but it would have caught this.